puppet/modules/leap/manifests/logfile.pp

```diff
@@ -0,0 +1,26 @@
+#
+# make syslog log to a particular file for a particular process.
+#
+
+define leap::logfile($process=$title) {
+  $logfile = "/var/log/leap/${title}.log"
```

Comment:
Maybe to make things consistent you could use ${process} here, instead of ${title}?